lua/autorun/server/sv_restrict_propcore.lua

```diff
@@ -41,6 +45,26 @@ function restrictPropCoreFunctions()
             end
         end
     end
+    
+    for _, signature in pairs( adminOnlyFunctions ) do
+        if wire_expression2_funcs then
+            local oldFunc = wire_expression2_funcs[signature][3]
+
+            wire_expression2_funcs[signature][3] = function( self, ... )
+                if ( self.player:IsAdmin() ) then
+                    local isInBuildMode = self.player:GetNWBool("CFC_PvP_Mode", false) == false
+
+                    if( isInBuildMode ) then
```

Comment:
My mistake, admins won't be able to use these in PvP now, I think you can just get rid of the pvp check entirely since admins should basically always be able to use them (MIght have to ask Phatso about it)

Reply:
Alrighty, I'll leave it as it was before this edit for now and contact Phatso for a final decision